maze/action.go

```diff
@@ -31,6 +30,9 @@ func action(ctx *cli.Context) {
 	if config.Interactive {
 		defer termbox.Close()
 		interactive(maze, config.Format)
+	} else if config.GenImage {
+		termbox.Close()
+		maze.PrintImage(config.Output, config.Format)
 	} else {
 		termbox.Close()
 		maze.Print(config.Output, config.Format)
```

Comment:
Please move the `if config.GenImage` clause inside the `else` clouse.
The code looks like: `Interactive` or `GenImage` or `Default`. But actually, `Interactive` mode or not, and if not, close the termbox soon. And then, in `GenImage` mode, it uses `PrintImage` and `Print` otherwise.